src/utils/sanitise.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @notice copied from https://gist.github.com/simonrenoult/2fa79fec8f035b2d35cef0767363b227
+ * */
+
+import _ from 'lodash';
+
+export const sanitise = (object: object): object | string | null => {
```

Comment:
Ha! Saw Palla's comment on this and felt like the spelling inconsistency would drive me to insanity. Though I prefer the "z", very content with consistency between function and file name!